### PR TITLE
add pages

### PR DESCRIPTION
**Pull request by hfrauenkron**
modify App.js, add router, add Welcome-page and assemble with components
image

### CODE REVIEW

File: src/pages/Welcome.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "styled-components";
+import MainBackground from "../components/MainBackground";
```
Comment (jsmunsch) [inline src/pages/Welcome.js]: You import MainBackground but I don't see it being used anywhere, is this on purpse ?

Comment (hfrauenkron) [reply src/pages/Welcome.js]: It turned out to be useless component, I deleted it in the meantime.

File: src/components/Favourite.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  font-weight: medium;
+  font-size: 20px;
+  border: none;
+  color: #242623;
+  padding: 0px;
+`;
+
+const StyledFavouriteIcon = styled(FavouriteIcon)`
+  cursor: pointer;
+  filter: drop-shadow(2px 2px 2px #00000029);
```
Comment (marwinburesch) [inline src/components/Favourite.js]: `#00000029` looks a little strange. I'd also recommend using `rgba()` for a transparent shadow

Comment (hfrauenkron) [reply src/components/Favourite.js]: Copied it from XD's developer link... 🤷‍♀️

File: src/components/Vote.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        `}
+`;
+
+const StyledLikeCount = styled.span`
+  padding-left: 5px;
+  font-size: 20;
+  border: none;
+`;
+
+// let clicks = "0";
```
Comment (lkastenmeier) [inline src/components/Vote.js]: Don't forget to remove commented-out code before a commit

File: src/components/Favourite.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import styled, { css } from "styled-components";
+import FavouriteIcon from "../icons/FavouriteIcon";
+
+const WrapperDiv = styled.div`
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-weight: medium;
+  font-size: 20px;
+  border: none;
```
Comment (lkastenmeier) [inline src/components/Favourite.js]: Is the border: none really needed here?

**Pull request merged**